lib/compress/huf_compress.c

```diff
@@ -758,6 +758,9 @@ typedef struct {
     } wksps;
 } HUF_compress_tables_t;
 
+#define SUSPECT_INCOMPRESSIBLE_SAMPLE_SIZE 4096
+#define SUSPECT_INCOMPRESSIBLE_SAMPLE_RATIO 10  // Must be larger than 2
```

Comment:
You should add a `DEBUG_STATIC_ASSERT(SUSPECT_INCOMPRESSIBLE_SAMPLE_RATIO >= 2)`. It needs to be within a function body, so probably like L801.

Also, `//` comments are disallowed in C89, use `/* */` instead.